ui/components/app/signature-request-siwe/signature-request-siwe-header/signature-request-siwe-header.js

```diff
@@ -0,0 +1,71 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import AccountListItem from '../../account-list-item';
+import { I18nContext } from '../../../../contexts/i18n';
+import Tooltip from '../../../ui/tooltip';
+import InfoIcon from '../../../ui/icon/info-icon.component';
+import { SEVERITIES } from '../../../../helpers/constants/design-system';
+
+import PermissionsConnectHeader from '../../permissions-connect-header';
+
+export default function SignatureRequestSIWEHeader({
+  fromAccount,
+  domain,
+  isSIWEDomainValid,
+  subjectMetadata,
+}) {
+  const t = useContext(I18nContext);
+
+  return (
+    <div className="signature-request-siwe-header">
+      <PermissionsConnectHeader
+        iconUrl={subjectMetadata.iconUrl}
+        iconName={subjectMetadata.name}
+        headerTitle={t('SIWESiteRequestTitle')}
+        headerText={t('SIWESiteRequestSubtitle')}
+        siteOrigin={domain}
+        className={isSIWEDomainValid ? '' : 'bad-domain'}
+        rightIcon={
+          !isSIWEDomainValid && (
+            <Tooltip
+              position="bottom"
+              html={<p>{t('SIWEWarningBody', [domain])}</p>}
```

Comment:
`SIWEWarningBody` is missing from the translations file  
<img width="407" alt="Screenshot 2022-05-26 at 4 33 42 PM" src="https://user-images.githubusercontent.com/20778143/170585953-0e4a9716-f374-457d-a85b-da969b8dd21e.png">

Reply:
Renamed, but didn't update everywhere! Updated.